core/src/main/java/org/keycloak/representations/oidc/OIDCClientRepresentation.java

```diff
@@ -293,6 +299,30 @@ public void setIdTokenEncryptedResponseEnc(String idTokenEncryptedResponseEnc) {
         this.id_token_encrypted_response_enc = idTokenEncryptedResponseEnc;
     }
 
+    public String getAccessTokenSignedResponseAlg() {
+        return access_token_signed_response_alg;
+    }
+
+    public void setAccessTokenSignedResponseAlg(String accessTokenSignedResponseAlg) {
+        this.access_token_signed_response_alg = accessTokenSignedResponseAlg;
+    }
+
+    public String getAccessTokenEncryptedResponseAlg() {
+        return id_token_encrypted_response_alg;
```

Comment:
I think it might be `access_token_encrypted_response_alg`. 
`JwtTokenEncryptionTest.java` might not find it because it uses the same algorithms for both Access Token and ID Token.